packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -158,9 +160,19 @@ export async function emailInputValidationAndAccountCheck(
       getString('new-user-email-validate')
     : 'Email is not valid';
 
-  const accountExistsMsg = getString
-    ? getString('new-user-existing-account-sign-in')
-    : `You already have an account, <a>Sign in</a>`;
+  const accountExistsMsg = (
+    <Localized
+      id="new-user-existing-account-sign-in"
+      elems={{ a: <a href={signInURL}></a> }}
+    >
+      <>
+        You already have an account,{' '}
```

Comment:
Nit: This is supposed to be a period not a comma, according to figma

![period](https://user-images.githubusercontent.com/1844554/128792757-539ac441-0760-440f-afb9-35f7d4df7c60.png)

Reply:
I noticed that too (yet somehow I forgot to change it).

This string was already here before my patch, so I think I'll have to change its localization ID to ensure the period is used in all of the string's localized versions.